frontend/resourceadm/components/ResourceNameAndId/ResourceNameAndId.tsx

```diff
@@ -107,17 +110,17 @@ export const ResourceNameAndId = ({
     <>
       <Paragraph size='small'>{text}</Paragraph>
       <Label className={classes.label} size='small'>
-        Ressursnavn (Bokmål)
+        {t('resourceadm.dahboard_resource_name_and_id_resource_name')}
       </Label>
       <div className={classes.textfieldWrapper}>
         <TextField
           value={title}
           onChange={(e) => handleEditTitle(e.target.value)}
-          aria-label='Ressursnavn (Bokmål)'
+          aria-label={t('resourceadm.dahboard_resource_name_and_id_resource_name')}
```

Comment:
Ref. prev comment. :)
```suggestion
      id="your-text-field-id"
```